cadquery/cq.py

```diff
@@ -3941,6 +3942,22 @@ def text(
         *  if combine is true, the value is combined with the context solid if it exists,
            and the resulting solid becomes the new context solid.
 
+        Examples::
+
+            cq.Workplane().text("CadQuery",5,1)
+
+        Specify the font (name), and kind to use an installed system font::
+
+            cq.Workplane().text("CadQuery",5,1,font="Liberation Sans Narrow", kind="italic")
```

Comment:
```suggestion
            cq.Workplane().text("CadQuery", 5, 1, font="Liberation Sans Narrow", kind="italic")
```

Reply:
I'll try to remember to run the black formatter next time.